Client/qtTeamTalk/generatettfiledlg.cpp

```diff
@@ -181,7 +181,7 @@ void GenerateTTFileDlg::slotSaveTTFile()
     QString start_dir = ttSettings->value(SETTINGS_LAST_DIRECTORY, QDir::homePath()).toString();
 
     QString filename = QFileDialog::getSaveFileName(this, tr("Save File"),
-        start_dir, tr("%1 File (*%1)").arg(TTFILE_EXT));
+        start_dir+"/"+m_hostentry.ipaddr+"_"+m_hostentry.username+".tt", tr("%1 File (*%1)").arg(TTFILE_EXT));
```

Comment:
Shouldn't we use "username@ipaddress_port.tt" instead? 
QString defaultfilename = QString("%1%2%3@%4_%5).arg(start_dir).arg(QDir::separator()).arg(m_hostentry.username).arg(m_hostentry.ipaddr).arg(m_hostentry.tcpport);